tortoise/models.py

```diff
@@ -1535,46 +1535,46 @@ def describe(cls, serializable: bool = True) -> dict:
 
             Each field is specified as defined in :meth:`tortoise.fields.base.Field.describe`
         """
+        fields: dict[str, list[dict]] = {
```

Comment:
Does it really improve performance? What is the impact? I think if you want to make a point that this improves performance, you need to add a benchmark as a separate PR and then provide the comparison. 

My honest opinion is that this change makes very readable code unreadable.

Reply:
Readable is more important.